model-api-gen-gradle-test/typescript-generation/build.gradle.kts

```diff
@@ -27,3 +28,17 @@ tasks.register<NpmTask>("packJsPackage") {
     dependsOn("npm_run_build")
     args.set(listOf("pack", "--pack-destination", "build"))
 }
+
+val updateModelClient = tasks.create<NpmTask>("updateModelClient") {
```

Comment:
Why is the `model-client` built by `model-api-gen-gradle-test/kotlin-generation`?
Related to https://github.com/modelix/modelix.core/pull/319/files#r1400396250